rector.php

```diff
@@ -64,38 +62,34 @@
 use Utils\Rector\RemoveErrorSuppressInTryCatchStmtsRector;
 use Utils\Rector\UnderscoreToCamelCaseVariableNameRector;
 
-return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->sets([
-        SetList::DEAD_CODE,
-        LevelSetList::UP_TO_PHP_81,
+return RectorConfig::configure()
+    ->withPhpSets(php81: true)
+    ->withPreparedSets(deadCode: true)
+    ->withSets([
         PHPUnitSetList::PHPUNIT_CODE_QUALITY,
         PHPUnitSetList::PHPUNIT_100,
-    ]);
-
-    $rectorConfig->parallel(120, 8, 10);
-
-    // Github action cache
-    $rectorConfig->cacheClass(FileCacheStorage::class);
-    if (is_dir('/tmp')) {
-        $rectorConfig->cacheDirectory('/tmp/rector');
-    }
-
+    ])
+    ->withParallel(120, 8, 10)
+    ->withCache(
+        // Github action cache or local
+        is_dir('/tmp') ? '/tmp/rector' : sys_get_temp_dir() . '/rector',
```

Comment:
I just make quick fix on develop for fallback `null` on local https://github.com/codeigniter4/CodeIgniter4/commit/86480f498af77afff0482194f5f9207ceb2685ee to use default value from rector if `/tmp` not exists.

https://github.com/rectorphp/rector-src/blob/8f0a09cc249e2725936a0eff3dd2651ebb10881a/config/config.php#L27